src/sif/autocost.cc

```diff
@@ -291,11 +296,12 @@ class AutoCost : public DynamicCost {
 public:
   VehicleType type_; // Vehicle type: car (default), motorcycle, etc
   std::vector<float> speedfactor_;
-  float density_factor_[16]; // Density factor
-  float highway_factor_;     // Factor applied when road is a motorway or trunk
-  float alley_factor_;       // Avoid alleys factor.
-  float toll_factor_;        // Factor applied when road has a toll
-  float surface_factor_;     // How much the surface factors are applied.
+  float density_factor_[16];   // Density factor
+  float highway_factor_;       // Factor applied when road is a motorway or trunk
+  float alley_factor_;         // Avoid alleys factor.
+  float toll_factor_;          // Factor applied when road has a toll
+  float surface_factor_;       // How much the surface factors are applied.
+  float living_street_factor_; // Avoid living streets factor.
```

Comment:
Added as an autocost-only member for now. Maybe should be moved to DynamicCost. Not sure which profiles should also be affected.